src/viking.js

```diff
@@ -44,7 +44,8 @@ class Soldier {
     super.receiveDamage(Damage);
     if(this.health > 0 ){ 
         return `A Saxon has received  ${Damage} points of damage`;
```

Comment:
failing this test "should return "A Saxon has received DAMAGE points of damage", if the Saxon is still alive" because you have 2 spaces between "received" and "Damage" and you should have one:
```
return `A Saxon has received ${Damage} points of damage`;
```